test/gst-vaapi/transcode/transcoder.py

```diff
@@ -11,44 +11,171 @@
 @slash.requires(have_gst)
 @slash.requires(*have_gst_element("vaapi"))
 @slash.requires(*have_gst_element("checksumsink2"))
+@platform_tags(ALL_PLATFORMS)
 class TranscoderTest(slash.Test):
+  requirements = dict(
+    decode = {
+      "avc" : dict(
+        sw = (ALL_PLATFORMS, have_gst_element("avdec_h264"), ("h264parse ! avdec_h264", None)),
+        hw = (AVC_DECODE_PLATFORMS, have_gst_element("vaapih264dec"), ("h264parse ! vaapih264dec", None)),
+      ),
+      "hevc-8" : dict(
+        sw = (ALL_PLATFORMS, have_gst_element("avdec_h265"), ("h265parse ! avdec_h265", None)),
+        hw = (HEVC_DECODE_8BIT_PLATFORMS, have_gst_element("vaapih265dec"), ("h265parse ! vaapih265dec", None)),
+      ),
+      "mpeg2" : dict(
+        hw = (MPEG2_DECODE_PLATFORMS, have_gst_element("vaapimpeg2dec"), ("mpegvideoparse ! vaapimpeg2dec", None)),
+      ),
+      "mjpeg" : dict(
+        hw = (MPEG2_DECODE_PLATFORMS, have_gst_element("vaapijpegdec"), ("jpegparse ! vaapijpegdec", None)),
+      ),
+      "vc1" : dict(
+        hw = (VC1_DECODE_PLATFORMS, have_gst_element("vaapivc1dec"), ("'video/x-wmv,profile=(string)advanced',width={width},height={height},framerate=14/1 ! vaapivc1dec", None)),
+      ),
+    },
+    encode = {
+      "avc" : dict(
+        sw = (ALL_PLATFORMS, have_gst_element("x264enc"), ("x264enc ! hvideo/x-h264,profile=main ! 264parse", "h264")),
```

Comment:
There's a typo here:  `hvideo/x-h264,profile=main` should be `video/x-h264,profile=main`

Reply:
thanks, fixed it